packages/tailwind/package.json

```diff
@@ -25,10 +25,9 @@
   "private": false,
   "license": "MIT",
   "dependencies": {
-    "@kaizen/design-tokens": "^10.3.16"
+    "@kaizen/design-tokens": "workspace:*"
```

Comment:
I think we can only do this with peers or devDeps, cause wouldn't the dep be published like this and consumers wouldn't get the right version?

Reply:
>This feature allows you to depend on your local workspace packages while still being able to publish the resulting packages to the remote registry without needing intermediary publish steps - your consumers will be able to use your published workspaces as any other package, still benefitting from the guarantees semver offers.

https://pnpm.io/workspaces#publishing-workspace-packages

:)

Reply:
So fancy